sdk/core/System.ClientModel/api/System.ClientModel.netstandard2.0.cs

```diff
@@ -83,6 +83,19 @@ public enum ClientErrorBehaviors
         Default = 0,
         NoThrow = 1,
     }
+    public partial class ClientLoggingOptions
+    {
+        public ClientLoggingOptions() { }
+        public System.Collections.Generic.IList<string> AllowedHeaderNames { get { throw null; } }
+        public System.Collections.Generic.IList<string> AllowedQueryParameters { get { throw null; } }
+        public bool? EnableLogging { get { throw null; } set { } }
```

Comment:
It's not clear what happens when EnableLogging is false but EnableMessageLogging is true. Maybe we prefer to have one property of an enum type. The enum would flags enum and would have the following values: None (sets the flags to all zero), Message, Content, All (all flags set). The user could then do, e.g.
options.EnableLogging = ClientLogging.None
options.EnableLogging = ClientLogging.Message | ClientLogging.Exceptions
options.EnableLogging = ClientLogging.All

... anyway, just an idea